### ISSUE TITLE
Weights gone missing?

### ISSUE THREAD

**Issue by masterfire6758** [antimatter15/alpaca.cpp#187]
I remember there were 30B and 65B weights on this repo at one point, now I just see 7B, having some trouble finding them if anyone could point me in the right direction that would be great :)

**Comment by Castaa** [comment]
magnet:?xt=urn:btih:F2BAEE27E31280C630093DAF0A7F4EC16EFCC126 for 30B

**Comment by JIBSIL** [comment]
Looks like it's been deleted by the repo owner.

You can find them in the repo history:
https://github.com/antimatter15/alpaca.cpp/blob/99f3908c515c73c7acbda307565234f33ec9738d/README.md

**Comment by masterfire6758** [comment]
> Looks like it's been deleted by the repo owner.
> 
> You can find them in the repo history: https://github.com/antimatter15/alpaca.cpp/blob/99f3908c515c73c7acbda307565234f33ec9738d/README.md

yeah, but the link was removed too, I was wondering if anyone knew where I could get the 65B weights or why they were removed
